lib/octokit/client/repositories.rb

```diff
@@ -177,6 +177,19 @@ def delete_repository(repo, options = {})
       end
       alias :delete_repo :delete_repository
 
+      # Transfer repository
+      #
+      # Requires authenticated repository or organization owner of source and destination.
```

Comment:
I'm not sure I understand what this comment is saying. Would you clarify, please?

Reply:
Oh, I misunderstand that.
I tried transfer from user to organization.
And I saw this [docs](https://help.github.com/articles/transferring-a-repository-owned-by-your-organization/).
What kind of document should I write?

Reply:
How about just replacing the first line of the comment with:

> Transfer a repository owned by your organization

And then getting rid of the "requires authenticated...". If we link to the docs, then they can see the documentation in detail by following the link.